packages/fiori/src/DynamicPageTitle.hbs

```diff
@@ -51,6 +50,7 @@
     </div>
 
   {{/if}}
+  <slot name="{{subheadingSlotName}}"></slot>
```

Comment:
I noticed an unintended side effect due to the position of the slot.
Please review the case when the `snappedTitleOnMobile` slot is in place and fix the if expression accordingly.